clonelists/Sony - PlayStation 2.json

```diff
@@ -159,6 +159,16 @@
 			["Aqua Aqua - Wetrix 2", 1],
 			["Aquaqua", 1]
 		],
+	    	"American Idol": [
```

Comment:
I'm kinda oldschool because I'm doing it with... Notepad++ ^^; (or with the "Edit file" button of Github).
VSC downloaded though, I'll check those issues right away!